src/main/java/com/homework/lesson14/adapter/Iphone.java

```diff
@@ -0,0 +1,13 @@
+package com.homework.lesson14.adapter;
+
+public class Iphone {
+    private int price;
+
+    public Iphone() {
+        price = 100;
+    }
+
+    public void getPrice() {
```

Comment:
As it is getter is should return price value but not print result.
Please rename this method or change its signature.